### DISCUSSION TITLE
Any way to public resource files from orchid ( JS/ SASS, components, webpack.mix, etc )

### DISCUSSION THREAD

**Discussion by didix16** [orchidsoftware/platform#2349] [General]
Hi, I'm wondering if is it possible get someway orchid resource files to customize css thorugh sass, add new fields with its hotwire controllers and compile them to get full control.

I know there is orchid:publish command but that command only copies the compiled files. What I'm looking for is a similar command that copy the resource files.

Is there any guide to fully get the control of frontend or any way I can edit CSS and compile JS?

Thanks

**Comment by astersnake** [comment]
As far as I know, it's not possible, and either is a good idea. You will be outdated soon with your version of the resources, and it will break your project.

**Reply by didix16** [reply]
So there is no chance to override some template blade files nor change styles outside vendor folder? Maybe some overriding custom files could be implemented?

Thanks
